docs/customer-portal/cp-dialogs.md

```diff
@@ -6,7 +6,5 @@
 
 !!! note "Ended Status"
     "Ended" indicates a call is over, but the telephony switches are still tearing down the call, including the Real-time Transport Protocol (RTP) stream and any final signaling. After that, the server gathers data for billing.
-!!! note "Global Dialogs"
-    You can also see the status of all the calls for specific Customers in **Global :material-menu-right: Dialogs**.
 
-    Also, you can select and end the calls using **End Calls** and the calls are terminaed using the MI Termination.
+    Also, you can select and end the calls using **End Calls** and the calls are terminated using the MI Termination.
```

Comment:
They cannot end calls from CP 
Remove this part

Reply:
ask adam